backend/src/database.ts

```diff
@@ -39,10 +39,10 @@ export type dbPseudoPlayer = {
   solvedPuzzles: string[];
 };
 
-export type dbPlayerWithoutEloId = Omit<dbPseudoPlayer, "eloId">;
+export type dbPlayerWithoutIdToken = Omit<dbPlayer, "idToken">;
 
-// We should not send the eloIds to the client viewing the ranking.
-export type dbRanking = dbPlayerWithoutEloId[];
+// We should not send the idTokens to the client viewing the ranking.
```

Comment:
Should id tokens be treated as a secret? i.e., is it bad to send a player's id token to another player?

Here, we omitted elo ids because it was secret. If the id token are not secret, then we don't need this.

Reply:
done